README.md

```diff
@@ -1,8 +1,51 @@
-# embedded-berm-bot
+# NASA Lunabotics
 
-Branch for testing the Arduino code
+This is a competition that is hosted every year by NASA and other sponsors. The current obejctive of the competition is to build a bot that is able to traverse a simulated lunar enviroment (baring low gravity), at the end of this traversal it will need to then collect the lunar simulant *[BP-1](https://ares.jsc.nasa.gov/projects/simulants/bp-1.html)*, *[LHS-1](https://spaceresourcetech.com/products/lhs-1-lunar-highlands-simulant)*, etc. once collected this material is then transported to a deposition/berm building area. The rules, guidelines, and documentation can be found [here](https://www.nasa.gov/learning-resources/lunabotics-challenge/).
 
-## MoonDawgArduino
+## embedded-berm-bot
 
-This is the the code that will be run on the robot.
-Make sure to follow the README in the "Libraries" folder to properly run and compile the file in the Arduino IDE
+For all things Arduino and Arduino-esque on the SIU Lunabotics bot. 
+Due to the nature of how our bot is constructed we have seperated the logic in two distinct places. 
+<details>
+  <summary>Raspberry Pi/Master</summary>
```

Comment:
raspberry pi?

Reply:
yeah i just dont understand the inclusion in the readme

Reply:
Somewhat true, prob can have most of it removed. But wanted to give context to how things were working on the Arduino

Reply:
I would not use a slash to describe it as both. Instead, maybe put "Raspberry Pi - Master" or "Raspberry Pi - 'Master'" something like that

Reply:
Done